### ISSUE TITLE
Treasury missing allocate buttons.

### ISSUE THREAD

**Issue by johnkslg** [StrangeLoopGames/EcoIssues#10577]
no allocate\tax(:P suggestion)\dump buttons
![default](https://user-images.githubusercontent.com/4980243/50826935-1c9def00-134e-11e9-808c-51a3f72fc862.png)

**Comment by D3nnis3n** [comment]
I'd love to see the tax button ;)

**Comment by Deantwo** [comment]
Did we get the Tax button? 😮

**Comment by elfl0rd** [comment]
No :(

**Comment by Deantwo** [comment]
No **Tax** button, and negative allocation is being fixed? I guess we can pay the treasury now, since bank accounts, so it isn't all bad.

But there goes the last power of the world leader I guess. At leadt until v9.0's civics.

But I might be one of the few hoping that world leader (or civic titles) will get the power to unclaim land as well as take people's mony. People would think more carefully about who should be world leader (or other titles) if the elected leader could steal everyone's money and land.
With civics of course it would be nice to vote on which titles had this power, amount of power.

TL;DR: Need tax button, world leader is becoming useless, and v9.0 hype rant.
